static/app/components/events/interfaces/performance/spanEvidenceKeyValueList.spec.tsx

```diff
@@ -51,6 +52,12 @@ describe('SpanEvidenceKeyValueList', () => {
       expect(
         screen.getByTestId('span-evidence-key-value-list.transaction')
       ).toHaveTextContent('/');
+      expect(
+        screen.getByTestId('span-evidence-key-value-list.transaction').querySelector('a')
+      ).toHaveAttribute(
+        'href',
+        `/organizations/org-slug/performance/summary/?project=123&referrer=performance-transaction-summary&transaction=%2F&unselectedSeries=p100%28%29`
```

Comment:
@mjq-sentry fyi this referrer is not correct, I didn't realize that `transactionSummaryRouteWithQuery` would add it.
I think we should add a referrer prop, to `transactionSummaryRouteWithQuery`. I can make that PR quickly.

Reply:
It's okay, I went through the exact same thought process 😅 Came to the same conclusion in the end too - I think it's (somewhat surprisingly) correct as is.